src/utils/configurations/checkNumeratorMappings.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns 'false' if all numerators lack data IDs
+ * (and 'true' if any one has been mapped to a data ID)
+ */
+export const checkNumeratorMappings = (configurations) => {
+    return configurations?.numerators.some((numerator) =>
```

Comment:
do you want the optional chaining for `?.some` (if configurations?.numerators is undefined then `some` won't work)?

Reply:
I actually prefer it this way: if `configurations` is undefined, the whole expression will evaluate to `undefined` which is safe (`.some()` won't throw) ~ since I know `configurations` comes from some context and fetching logic, I anticipate that might be undefined, BUT if `configurations` _is_ defined, I kinda like to assert that `numerators` is indeed a property on it, since that's pretty important. If `numerators` is undefined, I'd prefer to have the error and "fail early" to show that something is wrong, so the underlying cause can be fixed